neo/Persistence/LevelDB/Store.cs

```diff
@@ -0,0 +1,69 @@
+using Neo.IO.Data.LevelDB;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Neo.Persistence.LevelDB
+{
+    public class Store : IStore
```

Comment:
I think we should use different names for different implementations. Like LevelDBStore / MemoryStore.

Reply:
In the near future, we will have only `MemoryStore`. Other stores will be in plugins. So we can rename it when we remove the `LevelDB`.

Reply:
I see, but can't we call it Memory Store just to be clear? Because now we know it happens on memory, but in the future we can have new developers and they might not get it (it is not common for storage to be put on memory). Even if we only have one Store, saying that is a "memory storage" could be very helpful.

Reply:
But they are in different namespaces.

`Neo.Persistence.LevelDB.Store`, `Neo.Persistence.Memory.Store`

They look good. But if we rename them:

`Neo.Persistence.LevelDB.LevelDBStore`, `Neo.Persistence.Memory.MemoryStore`

This seems to be very repetitive.

Reply:
Why do you think it is repetitive? It looks ok to me.
Also, VS gets completely crazy because of that, it cannot find the correct Store in neo-cli. Maybe it would work if I have used the full path, the issue is that by using 'Store', it doesn't suggest any imports.